VERSIONS.md

```diff
@@ -0,0 +1,89 @@
+# 3.0.0 Alpha
```

Comment:
Pretty sure we're missing entries here or have never updated them in master, you added a lot more to the demo than `included demo` justifies :)

We should also add a

```
# 4.0.0
- Conversion to Godot 4
```